### ISSUE TITLE
Make otcfindunsupportednodes more general

### ISSUE THREAD

**Issue by josephwb** [OpenTreeOfLife/otcetera#2]
It seems ```otcfindunsupportednodes``` _only_ works when ottIDs are present. It would be convenient if this were not the case. For example, all of our test cases do not have such information.
E.g. (arbitrary) test "attachresolved":
```
taxonomy:
(A,B,C,D,E,F,G,H,I)life;
input trees:
((A,B),(C,D));
((A,C),(B,D));
((A,D),(E,F,G,H,I));
synth tree:
(((A,B),(C,D)),(E,F,G,H,I));
```
All of the information to conduct the test is there, it is just not expressed with ```ott####```. The error I get on the above is:
```
josephwb@WOPR:~/Desktop/treemachine/test-synth/attachresolved$ otcfindunsupportednodes taxonomy.tre expected.tre tree*
2015-03-25 11:46:23,511 INFO  [default] reading "taxonomy.tre"...
ERROR. Exiting due to an exception:
Error found "A" Expecting a name for a taxon to end with an ott##### where the numbers are the OTT Id. At line 1, column 2, filepos 2 of "taxonomy.tre"
```

**Comment by mtholder** [comment]
yeah. all of the tip matching tools in otcetera is based on ott ID being the trailing numeric suffix in every label.
It certainly would be possible to relax this, but I doubt that I'll have time anytime soon. In the interim, the test could be altered to add numbers. There is no validation of the label with OTT (using webservices or the dump of the taxonomy), so it is easy to fool it.
